src/components/Henvisning1963/Henvisning1963.tsx

```diff
@@ -0,0 +1,56 @@
+import { FormattedMessage } from 'react-intl'
+
+import { BodyLong, Button, Heading, HStack, VStack } from '@navikt/ds-react'
+
+import { Card } from '../common/Card'
+import { externalUrls } from '@/router'
+import { wrapLogger } from '@/utils/logging'
+
+interface IProps {
+  onAvbryt: () => void
```

Comment:
Jeg forstod ikke hvorfor den tar onAvbryt som prop, men det forklares kanskje av seg selv etterhvert

Reply:
Lurte litt på det selv. Tenkte kanskje at disse feilene ikke trengte å være egne URLer? Men at heller start kunne kondisjonelt vise de?

Altså i `Step0` ha logikk som enten returerer en av feilkomponentene eller `<Start />`. I JIRA står det at disse skal være egne URLer, men vet ikke helt hvorfor.

Reply:
Det var ønskelig med egne url'er for enklere Amplitude måling.
Videre i stegvisning f.eks har vi én feilside (uavhengig av om det kommer feil fra /inntekt eller /person) fordi det var ikke ønskelig å skille på dem i Amplitude.